recipes/core/rootfs-skel.yaml

```diff
@@ -0,0 +1,20 @@
+# Creates basic structure for a Linux system. This uses the merged /usr approach.
+
+environment:
```

Comment:
I would suggest to use `privateEnvironment` here. It's not needed by upstream recipes.